### PR TITLE
Update doc: Troubleshooting > Monitoring Service Health

### PR DESCRIPTION
**Pull request by rina23q**
## Proposed changes

Coincidentally, I found the doc was not updated when we switched from RFC3339 to Unix time in the service status payload.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
https://github.com/thin-edge/thin-edge.io/issues/2337

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by reubenmiller — CHANGES_REQUESTED** [review]
Actually both unix and rfc-3339 time formats are supported, and are controlled by configuration (though the default is `unix`):

```
tedge config set service.timestamp_format rfc-3339
```

**Comment by didier-wenzek** [comment]
> Actually both unix and rfc-3339 time formats are supported, and are controlled by configuration (though the default is `unix`):
> 
> ```
> tedge config set service.timestamp_format rfc-3339
> ```

The default being `unix`, I would apply the changes proposed by @rina23q but telling that the format can be changed with the tedge config setting for `service.timestamp_format`.

File: docs/src/operate/troubleshooting/monitoring-service-health.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ For example, `tedge-mapper-c8y` publishes a message on topic `te/device/main/ser
 |----------|----------------------------------------------------|
 | `pid`    | Process ID of the service                          |
 | `status` | Service status. Possible values are `up` or `down` |
-| `time`   | Timestamp in RFC3339 format                        |
+| `time`   | Timestamp in Unix time format                      |
```
Comment (didier-wenzek) [inline docs/src/operate/troubleshooting/monitoring-service-health.md:40]: Timestamp in the  time format specified by `tedge config` for `service.timestamp_format`: either  `unix` or `rfc-3339`.

File: docs/src/operate/troubleshooting/monitoring-service-health.md [hunk 2]
```diff
@@ -28,16 +28,16 @@ If the response is not received then most likely the service is down, or not res
...
 
-| Property | Description                                        |
-|----------|----------------------------------------------------|
-| `pid`    | Process ID of the service                          |
-| `status` | Service status. Possible values are `up` or `down` |
-| `time`   | Timestamp in RFC3339 format                        |
+| Property | Description                                                                                           |
+|----------|-------------------------------------------------------------------------------------------------------|
+| `pid`    | Process ID of the service                                                                             |
+| `status` | Service status. Possible values are `up` or `down`                                                    |
```
Comment (rina23q) [inline docs/src/operate/troubleshooting/monitoring-service-health.md:39]: My editor adds the spaces automatically, hope no one cares

File: docs/src/operate/troubleshooting/monitoring-service-health.md [hunk 3]
```diff
@@ -28,16 +28,16 @@ If the response is not received then most likely the service is down, or not res
...
+| `time`   | Timestamp in either Unix or RFC-3339 format. Configurable by tedge config as service.timestamp_format |
```
[the remark this answers is not in the document]
Comment (rina23q) [reply docs/src/operate/troubleshooting/monitoring-service-health.md]: Done

File: docs/src/operate/troubleshooting/monitoring-service-health.md [hunk 4]
```diff
@@ -28,16 +28,16 @@ If the response is not received then most likely the service is down, or not res
...
-| Property | Description                                        |
-|----------|----------------------------------------------------|
-| `pid`    | Process ID of the service                          |
-| `status` | Service status. Possible values are `up` or `down` |
-| `time`   | Timestamp in RFC3339 format                        |
+| Property | Description                                                                                             |
+|----------|---------------------------------------------------------------------------------------------------------|
+| `pid`    | Process ID of the service                                                                               |
+| `status` | Service status. Possible values are `up` or `down`                                                      |
+| `time`   | Timestamp in either Unix or RFC-3339 format. Configurable by tedge config as `service.timestamp_format` |
```
Comment (reubenmiller) [inline docs/src/operate/troubleshooting/monitoring-service-health.md:40]:
```suggestion
| `time`   | Timestamp in either Unix or RFC-3339 format. Configurable by tedge config setting  `service.timestamp_format` |
```

**Review by reubenmiller — APPROVED** [review]
Just one minor tweak on the phrasing about the setting.

https://github.com/thin-edge/thin-edge.io/pull/2855/files#r1589155917

**Comment by rina23q** [comment]
> Just one minor tweak on the phrasing about the setting.
> 
> https://github.com/thin-edge/thin-edge.io/pull/2855/files#r1589155917

Applied the change

**Comment by reubenmiller** [comment]
@rina23q looks great. My comments have been addressed :)

**Pull request merged**